generate_openj9_scc.sh

```diff
@@ -70,9 +44,21 @@ function check_to_download_tomcat() {
 
         # Downloading tomcat
         if curl --fail -o "${DOWNLOAD_PATH_TOMCAT}"/tomcat.tar.gz "${TOMCAT_DWNLD_URL}"; then
-            # Extracting tomcat
-            tar -xvzf "${DOWNLOAD_PATH_TOMCAT}"/tomcat.tar.gz -C "${INSTALL_PATH_TOMCAT}" --strip-components=1
+            # Verifying checksum
+            echo "${TOMCAT_CHECKSUM} *${DOWNLOAD_PATH_TOMCAT}/tomcat.tar.gz" | sha512sum -c
+            if [ $? != 0 ]; then
```

Comment:
Please use `if cmd; then` format and not check for `$?`

Reply:
Updated the script with changes